posthog-android/src/main/java/com/posthog/android/replay/PostHogReplayIntegration.kt

```diff
@@ -203,22 +203,21 @@ public class PostHogReplayIntegration(
 
     private val onTouchEventListener =
         OnTouchEventListener { motionEvent ->
-            if (!isSessionReplayEnabled) {
-                return@OnTouchEventListener
-            }
-            motionEvent.eventTime
-            val timestamp = config.dateProvider.currentTimeMillis()
-            when (motionEvent.action.and(MotionEvent.ACTION_MASK)) {
-                MotionEvent.ACTION_DOWN -> {
-                    generateMouseInteractions(timestamp, motionEvent, RRMouseInteraction.TouchStart)
+            try {
```

Comment:
I don't see how this could throw but I recall a lot of strange dispatchTouchEvent  errors in the past so let's just be super careful.